cmd/context/utils.go

```diff
@@ -144,9 +145,10 @@ func LoadManifestWithContext(ctx context.Context, opts ManifestOptions) (*model.
 	}
 
 	ctxOptions := &ContextOptions{
-		Context:   ctxResource.Context,
-		Namespace: ctxResource.Namespace,
-		Show:      true,
+		Context:       ctxResource.Context,
+		Namespace:     ctxResource.Namespace,
+		Show:          true,
+		DockerDesktop: os.Getenv(model.OktetoOriginEnvVar) == model.OktetoDockerDesktopOrigin,
```

Comment:
is this needed if we are already doing?
https://github.com/okteto/okteto/pull/2546/files#diff-c1b929a650f782cf212e9b4b0ced5a7a298598bcb7473be505e7af1f06d6cf30R85

Reply:
Following with my previous comment and with what Pablo said, this should not be necessary. We store it as an envvar and retrieve it as an envvar

Reply:
We need it because on `okteto up` there is no other way of knowing if it's coming from the docker desktop extension